Weather/Views/TimedForecastCellView.swift

```diff
@@ -9,21 +9,62 @@
 import UIKit
 
 class TimedForecastCellView: UICollectionViewCell {
-    let timeLabel = UILabel()
-    let temperatureLabel = UILabel()
+    private let timeLabel = UILabel()
+    private let temperatureLabel = UILabel()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        timeLabel.text = "2 AM"
```

Comment:
You don't need the default value here as it is does make any sense to user if for some reason you couldn't populate it with correct data or either you did some mistake and now user will suddenly see "2 AM".

Also consider adding some sort of "Day" model to incapsulate logic of converting data from 12 to 24 base, as well as from unix timestamp for example as it is the common practice to send the time as a `unix timestamp` or `RFC 3339  timestamp`

The same applies to the temperature representation. Incapsulate it with another data object.

Reply:
Make sure to fix all the similar issues as well not only in this particular class.